app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_people = [
+        Person(name=human["name"], age=human["age"])
+        for human in people
```

Comment:
This can fit on one line easy

Reply:
in both cases flake8 gives an error that the string length is too long